### PR TITLE
feat: add spots comments

### PR DESCRIPTION
**Pull request by antoinechalifour**

### CODE REVIEW

File: src/domain/model/Comment.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export class Comment {
+  public constructor(
+    public id: string,
+    public authorId: string,
+    public spotId: string,
+    public body: string
+  ) {}
```
Comment (francoischalifour) [inline src/domain/model/Comment.ts:8]: Do we plan to add a date in another PR?

**Review by francoischalifour — APPROVED** [review]
Regarding backend errors, should we rely on status codes rather than sentences? How are we going to deal with them on the frontend?

File: src/domain/model/Comment.ts (hunk 1 above)
Comment (antoinechalifour) [reply src/domain/model/Comment.ts:8]: I guess we can add it in this PR

**Comment by antoinechalifour** [comment]
> Regarding backend errors, should we rely on status codes rather than sentences? How are we going to deal with them on the frontend?

We'll think about it later and implement it in another PR

**Pull request merged**